acs-integration-tests/src/test/java/com/ge/predix/integration/test/ACSMeteringIT.java

```diff
@@ -174,6 +191,30 @@ private Double getEntitlementUsageByFeatureId(final String featureId, final Stri
         return entitlement.getCurrentUsedAmount();
     }
 
+    private double getSteadyEntitlementUsage(final String featureId, final String subscriptionId)
+            throws Exception {
+        LOGGER.info("Entered Meter Steady Stage");
+
+        double entitlementUsage_1 = 0.0;
+        double entitlementUsage_2 = 0.0;
+
+        for( int i = 0; i < MAX_STEADY_ITERATIONS; i++ ) {
+            entitlementUsage_1 = getEntitlementUsageByFeatureId(featureId, subscriptionId);
+
+            Thread.sleep(10000);
```

Comment:
I would consider making this sleep time a constant (i.e. private static final) or perhaps even environment variable configurable with a default. e.g.

```
@Value("${STEADY_STATE_SLEEP_MS:10000}")
private int steadyStateSleepMs;
```